src/index.js

```diff
@@ -55,7 +55,7 @@ export default class ServiceWorkerPlugin {
         // Hijack the original module
         if (result.resource === runtimePath) {
           const data = {
-            scriptURL: this.options.publicPath + this.options.filename,
+            scriptURL: (this.options.publicPath + this.options.filename).replace(/^\/{2,}/, '/'),
```

Comment:
Merge conflicts.

And, isn't this problem better solved by using `path.normalize`?

Or, even better, by using `path.join` instead if string concat.

Reply:
Great idea!